mapping_form.html

```diff
@@ -0,0 +1,150 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <title>MANYC Airtable-Gateway Mapping Form</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <style>
+    </style>
+</head>
+
+<body>
+
+    <div class="container">
+        <h1>Welcome to MANYC Airtable-Gateway</h1>
+        <h5>Please enter in your group's Airtable Columns that can correlate with our columns show below...</h5>
+        <form class="form-horizontal" action="api/setMapping" method="POST">
+            <div class="form-group">
+                <label class="control-label col-sm-2" for="status">Status:</label>
+                <div class="col-sm-10">
+                    <input name= "status" type="status" class="form-control" id="status" placeholder="Enter Your Status Column"
+                        name="status" value="">
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="control-label col-sm-2" for="id">Id:</label>
+                <div class="col-sm-10">
+                    <input type="id" class="form-control" id="id" placeholder="Enter Your Id Column" name="id">
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="control-label col-sm-2" for="supportType">Support Type:</label>
+                <div class="col-sm-10">
+                    <input type="supportType" class="form-control" id="supportType"
```

Comment:
Are arbitrary strings for `type` recommended by Bootstrap? The `type` attribute has a fairly [defined list of values](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#%3Cinput%3E_types) that are interpreted by the browser for styling and validation. Generally `type="text"` is what we'd want for text fields.

Reply:
Good catch. Fixed 👍